platform-api/src/routers/platform.py

```
@@ -0,0 +1,99 @@
+        site=site,
+        status=data.get("latest_stage", {}).get("status", "unknown"),
+        url=data.get("url"),
+        created_on=data.get("created_on"),
+        environment=data.get("environment", "production")
+    )
+
+@router.post("/rebuild", response_model=RebuildResponse)
+async def rebuild(
+    site: str = Query(..., description="Target site to rebuild"), 
+    settings: WorkerSettings = Depends(get_settings), 
+    _ = Depends(verify_admin_api_key)
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)